replace_words.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+import sys
+import itertools
+import os
+import argparse
+
+
+class WordsModifier:
+    default_input_file = 'file1.txt'
+
+    def __init__(self, input_file, output_file, from_list_arr, to_list_arr):
+        self.input_file = input_file

```

Comment:
I think you should use some default values for the parameters of the init.

also, provide some set and get for them <use properties in python>.